lms/templates/credit_notifications/credit_eligibility_email.txt

```diff
@@ -9,8 +13,7 @@ ${dashboard_link}
 
 ${_(u"Course credit can help you get a jump start on your university degree, finish a degree already started, or fulfill requirements at a different academic institution.")}
 
-${_(u'To get course credit, simply go to your edX dashboard and click the "Get Credit" button. No application, transcript, or grade report is required.')}
+${_(u'To get course credit, simply go to your edX dashboard and click the "Get Credit" button. After you purchase credit you will have access to your official academic transcript from Arizona State University.')}
```

Comment:
Hi @zubair-arbi, I don't think we can hard code ASU here, and may need to alter the text if we can't use configuration parameters at all for the provider name. Is provider name not something we can pull in here? cc'ed @pbaruah